src/Symfony/Component/Security/Core/Resources/translations/security.he.xlf

```diff
@@ -8,51 +8,51 @@
             </trans-unit>
             <trans-unit id="2">
                 <source>Authentication credentials could not be found.</source>
-                <target>Authentication credentials could not be found.</target>
+                <target>פרטי זיהוי לא נמצאו.</target>
```

Comment:
Could you double check this? 

Google translated this to "id not found"

Reply:
In Hebrew it’s hard to translate it
That’s the best

Reply:
Okey. Good. =)